testing/tests/redirects.test.js

```diff
@@ -676,7 +676,7 @@ const ZONE_REDIRECTS = [
   ["Marketplace", "Mozilla/بازار", "APIs", ["fa"]],
 ];
 
-ZONE_REDIRECT_URLS = [];
+let ZONE_REDIRECT_URLS = [];
```

Comment:
Shouldn't that be 
```suggestion
const ZONE_REDIRECT_URLS = [];
```
...for consistency? Or does it ever get redefined?

Reply:
Doesn't get redifined, just pushed to, so I think it can be const. I was just being cautious